src/Components/Shifting/ListView.tsx

```diff
@@ -63,190 +58,168 @@ export default function ListView() {
-                      <dt
-                        title={t("origin_facility")}
-                        className="flex items-center text-sm font-medium leading-5 text-secondary-500"
-                      >
-                        <CareIcon icon="l-plane-departure" className="mr-2" />
-                        <dd className="text-sm font-bold leading-5 text-secondary-900">
-                          {(shift.origin_facility_object || {}).name}
-                        </dd>
-                      </dt>
-                    </div>
-                    {careConfig.wartimeShifting && (
-                      <div className="sm:col-span-1">
-                        <dt
-                          title={t("shifting_approving_facility")}
-                          className="flex items-center text-sm font-medium leading-5 text-secondary-500"
-                        >
-                          <CareIcon icon="l-user-check" className="mr-2" />
-                          <dd className="text-sm font-bold leading-5 text-secondary-900">
-                            {
-                              (shift.shifting_approving_facility_object || {})
-                                .name
-                            }
-                          </dd>
-                        </dt>
-                      </div>
-                    )}
-                    <div className="sm:col-span-1">
-                      <dt
-                        title={t("assigned_facility")}
-                        className="flex items-center text-sm font-medium leading-5 text-secondary-500"
-                      >
-                        <CareIcon icon="l-plane-arrival" className="m-2" />
-
-                        <dd className="text-sm font-bold leading-5 text-secondary-900">
-                          {shift.assigned_facility_external ||
-                            shift.assigned_facility_object?.name ||
-                            t("yet_to_be_decided")}
-                        </dd>
-                      </dt>
-                    </div>
-
-                    <div className="sm:col-span-1">
-                      <dt
-                        title={t("last_modified")}
-                        className={
-                          "flex items-center text-sm font-medium leading-5 " +
-                          (dayjs()
-                            .subtract(2, "hours")
-                            .isBefore(shift.modified_date)
-                            ? "text-secondary-900"
-                            : "rounded bg-red-400 p-1 text-white")
-                        }
-                      >
-                        <CareIcon icon="l-stopwatch" className="mr-2" />
-                        <dd className="text-sm font-bold leading-5">
-                          {formatDateTime(shift.modified_date) || "--"}
-                        </dd>
-                      </dt>
-                    </div>
+    return data.map((shift: any) => (
```

Comment:
Can we reuse the `<Table/>` component in any way? `src/components/Common/components/Table.tsx`. I would prefer if we had a component that can handle the table view.

Reply:
@shivankacker  
Should i Proceed to work on these thing?. I saw PR#8934 regarding the UI design in shifiting.

Reply:
By Using the Table component . We need again redesign the showShifitngList inorder to return array of array as rows prop for Table component. Seems Like it would need to rework again if we use the table component

Reply:
@bodhish Would you confirm Either to continue to work on or Leave it

Reply:
@AnveshNalimela please continue working on this PR, we need the table layout instead of grid layout. https://github.com/ohcnetwork/care_fe/pull/8934 had some file and code restructures.

Reply:
> By Using the Table component . We need again redesign the showShifitngList inorder to return array of array as rows prop for Table component. Seems Like it would need to rework again if we use the table component

What about these..?